Snowcast/ResortsTableViewController.swift

```diff
@@ -1,21 +1,39 @@
+import PromiseKit
 import UIKit
+import CoreLocation
 
 class ResortsTableViewController: UITableViewController {
     
     let resortTableCell = "resortTableCell"
-    let showResortDetailSegue = "showResortDetailSegue"
-    
-    var resortService: ResortService?
-    var resorts: Array<ResortPreview> = []
+    var requestSnowData: RequestSnowData = RequestSnowData()
+    var resortService: ResortService = ResortService()
+    var locationService: UserLocation = UserLocation()
+    var selectedIndexPath : NSIndexPath?
+    var activityIndicator = UIActivityIndicatorView()
+
+    var resorts: Array<ResortPreview> = [] {
+        didSet {
+            self.tableView.reloadData()
+        }
+    }
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        self.resortService = ResortService()
         
-        self.resortService?.fetchResorts() { (let resorts) in
+        locationService.requestLocation()
+        activitiyView()
+        
+        tableView.registerNib(UINib(nibName: "ResortTableViewCell", bundle: nil), forCellReuseIdentifier: resortTableCell)
```

Comment:
Length Violation: Line should be 100 characters or less: currently 122 characters